docker-compose.yml

```diff
@@ -126,12 +130,13 @@ services:
     build:
       context: .
       dockerfile: rocky/Dockerfile
+      target: dev
       args:
         ENVIRONMENT: dev
     command: python3 manage.py runserver 0.0.0.0:80
     volumes:
-      - ./rocky:/app/rocky
-      - ./octopoes/octopoes:/app/rocky/octopoes
+      - ./rocky:/app/rocky:ro
```

Comment:
How are you supposed to run makemigrations if the volume is read only? Makemigrations won't be able to write the migration file. Same for the modules that use alembic.

Reply:
Migration files are usually made on the development machine and not inside the container, right?

Reply:
The Makefile targets often use the containers (this also holds for Katalogus/Bytes)

Reply:
Good point, `makemigrations` is a valid use case for having a writable volume. Do we revert the read-only change for all modules using Django or alembic?

Reply:
Cool!